hack/e2e-kind-cluster-setup.sh

```diff
@@ -25,18 +29,42 @@ setup_cluster() {
 push_local_image() {
     OCI_BIN="$OCI_BIN" IMAGE_REGISTRY="$IMG_REGISTRY" IMAGE_TAG="$IMG_TAG" make manifests
     OCI_BIN="$OCI_BIN" IMAGE_REGISTRY="$IMG_REGISTRY" IMAGE_TAG="$IMG_TAG" make img-build
-    kind load docker-image $IMG_REGISTRY/multus-dynamic-networks-controller:$IMG_TAG
+    if [[ $KUBEVIRT_PROVIDER != "external" ]]; then
```

Comment:
IMHO this makes no sense. The condition should be if the target is kind, use kind-based commands.